person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..9999)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    of_age?
```

Comment:
- [ ] Please make sure that your Public method can_use_services?  returns true in case person is of age or if person has permission from parents. You should have something like the example below :arrow_down:  

```
  def can_use_services? 
    of_age?() || @parent_permission
  end
```